### ISSUE TITLE
Stabilizers don't fit in PE Acrylic 3mm

### ISSUE THREAD

**Issue by konisteel** [keebio/iris-case#45]
Hey, so i laser cut the Iris PE Switch Plate Mx 3mm and after receiving it my stabilizers & the switches fall through the thumb area because the space it too large. The shop did not enlarge that area. How am i supposed to fit the stabilizers? I am going to handwire it so the stabs have to fit in the plate

https://github.com/user-attachments/assets/364dcd68-56ff-40d8-9d20-8b80f713b90c

 
![image](https://github.com/user-attachments/assets/6a7d9205-b812-4fa5-85d2-2fe4f750c20a)

Thanks for the help

**Comment by nooges** [comment]
The cutout around the 2u key for 3mm plate is large to accommodate the stabilizer wire/housing. Otherwise, the wire and housing would hit the underside of the plate. Hence, the 3mm plate isn’t geared towards handwiring and more for use with a PCB.

**Comment by konisteel** [comment]
Yea i figured that part out later after a lot of troubleshooting. Is it possible to use an adapter over there? The adapter will be 1.5mm and hold the switch & the stabs. Just a thought

**Comment by nooges** [comment]
I’ll have to check my files, but I may have already designed a 3D-printable adapter to hold the switch in-place.

**Comment by konisteel** [comment]
There is already a file to hold 2 1u switches but i want to use 1 2u
If you have the files, It would be a great help 🙏.
Thanks

**Comment by konisteel** [comment]
Did you fine the stl file?

**Comment by nooges** [comment]
I checked my Fusion 360 projects, and it doesn't look like I've designed a
2u adapter yet, but I can give it a go later this week.

**Comment by konisteel** [comment]
That Would help me out a bunch, and others who will use it :D 
Can't wait for the files and the new update

**Comment by nooges** [comment]
Yeah, I'm working something up right now and will do some test printing

**Comment by konisteel** [comment]
Great!

**Comment by konisteel** [comment]
Hey when can we expect the stl file :p

**Comment by konisteel** [comment]
[keyboard adapter.zip](https://github.com/user-attachments/files/17617488/keyboard.adapter.zip)

**Comment by nooges** [comment]
Turns out I did have something designed previously, see how this works out for you.
[Iris PE 2u Adapter for 3mm Plate.stl.zip](https://github.com/user-attachments/files/17649087/Iris.PE.2u.Adapter.for.3mm.Plate.stl.zip)

**Comment by nooges** [comment]
If recutting the plate is an option, you could go with the older-style cutout in iris-PE-switch-plate-mx-alps.dxf

**Comment by konisteel** [comment]
Hey how does the file work? I'm confused
![image](https://github.com/user-attachments/assets/55289449-ade5-4e64-8d4d-4ac71078e1e3)

**Comment by nooges** [comment]
It’s not particularly great with handwiring though, so I don’t really have a good solution for using the plate as-isOn Nov 6, 2024, at 10:35 AM, Konisteel ***@***.***> wrote:﻿
Hey how does the file work? I'm confused
image.png (view on web)

**Comment by konisteel** [comment]
Alright i'll receive the 3d printed adapter that i uploaded and check if everything works great. I'll keep you updated but so far if you use the top plate than a little sanding is necessary or else it'll fit on the plate no problem with normal mx switches and cherry stabs
